api_docs/api_specs.json

```diff
@@ -469,6 +469,56 @@
         "comment": "Response acknowledging the successful deletion of the config",
         "schema": "default_message.json"
       }
+    },
+    "bridge_update_check": {
+      "title": "Bridge update check",
+      "description": "Checks whether the bridge has an update available. Responds with bool value for available update and branch information",
+      "broadcast_allowed": true,
+      "http_method": null,
+      "sender": [
+        "Web Application",
+        "Toolkit"
+      ],
+      "receiver": [
+        "Bridge"
+      ],
+      "uri": {
+        "value": "bridge/update/check",
+        "var_name": "bridge_update_check"
+      },
+      "request": {
+        "comment": "Any payload. Response will mirror the payload sent.",
+        "schema": "api_empty_request.json"
+      },
+      "response": {
+        "comment": "Response payload contains bool value for update_available and branch information (old and new, if not newest)",
+        "schema": "api_bridge_update_response.json"
+      }
+    },
+    "Bridge_update_execute": {
+      "title": "Bridge update execute",
+      "description": "Execute update on bridge. No Response required, system reboots",
+      "broadcast_allowed": true,
+      "http_method": null,
+      "sender": [
+        "Web Application",
+        "Toolkit"
+      ],
+      "receiver": [
+        "Bridge"
+      ],
+      "uri": {
+        "value": "bridge/update/execute",
+        "var_name": "bridge_update_execute"
+      },
+      "request": {
+        "comment": "Any payload. Response will mirror the payload sent.",
```

Comment:
Shouldn't the user get some response indicating the operation was successful?

Reply:
User will receive response, indicating whether the update was successful or not.